source/UF/H-Levels.lagda

```diff
@@ -197,27 +193,25 @@ From univalence we can show that ℍ n is of level (n + 1), for all n : ℕ.
 
 \begin{code}
 
-equiv-preserves-hlevel : (n : ℕ) {X : 𝓤 ̇  } {Y : 𝓥 ̇  }
+equiv-preserves-hlevel : {n : ℕ} {X : 𝓤 ̇  } {Y : 𝓥 ̇  }
```

Comment:
Please replace " ̇  }" by " ̇ }" everywhere. There is a double space. We need only one space between the upper dot and the brace. Similarly with round brackets. In the other file as well.